pages/about.html

```diff
@@ -4,5 +4,32 @@
 layout: about
 home: false
 ---
-
-<h2>About Content Goes Here</h2>
\ No newline at end of file
+<div class="aboutPage">
+  <div class="section">
+    <div class="infoContainer">
+      <h1>Who is Code for America?</h1>
+      <div class="text">
+        Code for America is a non-partisan, non-political 501 organization founded in 2009 to address the widening gap between the public and private sectors in their effective use of technology and design. Part of what they do is lead the Code for America Brigade Network where over 25,000 active members are committed to making their local communities better. Leveraging on the Code for America name and the tools they provide improve the way in which we work.
+      </div>
+      <p class="learnMore">
+        <a href="">Learn more about Code for America <i class="fas fa-long-arrow-alt-right"></i></a>
+      </p>
+    </div>
+    <img class="cardImage" src="/images/proxy-img.png" alt="A sample proxy image">
+  </div>
+  <div class="section">
+    <div class="infoContainer">
+      <h1>Why join us?</h1>
+      <div class="text">
+        Code for Chicago operates like a pro-bono digital consultancy. In lieu of making decisions on behalf of the communities we serve, we operate shoulder-to-shoulder. When we start a new project we typically partner up with a local organization. This means we begin by identifying their needs, drafting a problem statement, and collaborating throughout the process. Since we aren't our users, this ensures we are effectively addressing the needs of the community and not operating from our own perspective as practitioners.
+      </div>
+      <div class="text">
+        Other than doing projects for a good cause, volunteers of various skill sets and levels join us to try out civic tech, network with like-minded people, and learn new skills.
+      </div>
+      <p class="learnMore">
+        <a href="">Learn more about Code for America <i class="fas fa-long-arrow-alt-right"></i></a>
```

Comment:
I believe this should say `Learn how the onboarding process works` instead

Reply:
updated